.ado/jobs/e2e-test.yml

```diff
@@ -90,8 +90,8 @@ jobs:
               displayName: yarn e2etest
               workingDirectory: packages/e2e-test-app
 
-            - script: yarn e2etest -u
-              displayName: Update snapshots
+            - script: npx jest --clearCache
```

Comment:
Whoops, this looks like a bad merge, whether or not we add the "clear jest cache" task, we still want the Update snapshots task to run on failure so we can get those updated snapshots.

Reply:
woops deleted the wrong task, fixed it! I have a separate PR to fix updating the snapshots (https://github.com/microsoft/react-native-windows/pull/11937) but needed to grab the snapshots for this PR